docs/content/manual/_incoming/_index.md

```diff
@@ -64,65 +64,62 @@ These are some potential new tests for next release.
-- [BUG] rd.cos.debugrw does not persist changes to /boot [#1388](https://github.com/harvester/harvester/issues/1388) owner:TachunLin p2
-- [BUG] nodes are not being deleted on the backend when a cluster is deleted through rancher [#1306](https://github.com/harvester/harvester/issues/1306) owner:TachunLin p2
-- [BUG] Harvester RKE2 cluster provisioning fails with air-gapped Rancher [#1247](https://github.com/harvester/harvester/issues/1247) owner:TachunLin p1
-- [BUG] Missing http proxy settings on rke2 and rancher pod [#1218](https://github.com/harvester/harvester/issues/1218) owner:TachunLin p1
-- [BUG] Guest agent install configuration is covered [#908](https://github.com/harvester/harvester/issues/908) owner:TachunLin p1
-- [Bug] Exclude OS root disk and partitions on forced GPT partition [#1382](https://github.com/harvester/harvester/issues/1382) owner:TachunLin p1
-- [Enhancement] Better instance config descriptions of Harvester RKE1 & RKE2 node driver [#1030](https://github.com/harvester/harvester/issues/1030) owner:TachunLin p2
-- [FEATURE]  toggle harvester node driver with the harvester global flag [#1465](https://github.com/harvester/harvester/issues/1465) owner:TachunLin p1
-- [FEATURE] better loadblancer config of Harvester cloud provider [#1435](https://github.com/harvester/harvester/issues/1435) owner:TachunLin p1
-- [FEATURE] Integration Cloud Provider for RKE1 with Rancher [#1396](https://github.com/harvester/harvester/issues/1396) owner:TachunLin p1
-- [FEATURE] Enhance the import experience to Rancher [#1330](https://github.com/harvester/harvester/issues/1330) owner:TachunLin p1
-- [FEATURE] Move Longhorn storage to another partition [#1316](https://github.com/harvester/harvester/issues/1316) owner:TachunLin p1
-- [FEATURE] Disk devices used for VM storage should be globally configurable [#1241](https://github.com/harvester/harvester/issues/1241) owner:TachunLin p1
-- [FEATURE] Host list should display the disk error message on table [#1167](https://github.com/harvester/harvester/issues/1167) owner:TachunLin p2
-- [FEATURE] Add network reachability detection from host for the VLAN network [#1476](https://github.com/harvester/harvester/issues/1476) owner:TachunLin, noahgildersleeve p1
-- [Feature] Support volume hot-unplug [#1401](https://github.com/harvester/harvester/issues/1401) owner:TachunLin p1
-- [Harvester] when a cluster member is added and then removed from the harvester cluster RBAC logged in as a Cluster owner all the members disappear [#1377](https://github.com/harvester/harvester/issues/1377) owner:TachunLin p2
-- [Task] Test Air gap with Rancher integration [#1052](https://github.com/harvester/harvester/issues/1052) owner:TachunLin p1
-- [feature] allow users to create cloud-config template on the VM creating page [#1433](https://github.com/harvester/harvester/issues/1433) owner:TachunLin p1
+- [#1476](https://github.com/harvester/harvester/issues/1476) 
```

Comment:
I think we don't need to list those issues added with markdown file.
They will be displayed twice with the same title in index page.

Reply:
I just removed the issue links in _index.md and make a new commit